### ISSUE TITLE
Disassembler incorrectly disassembles "66 90"

### ISSUE THREAD

**Issue by Symbai** [cheat-engine/cheat-engine#1067]
`66 90` is `xchg ax,ax` but CE disassembles it as `nop`. While it technically has the same effect as a NOP it causes the assembler to generate `90` on this so its missing a byte when disassembled => assembled and therefore should be fixed.

![image](https://user-images.githubusercontent.com/14368203/71673804-9f933300-2d71-11ea-9264-cee25685336b.png)

**Comment by cheat-engine** [comment]
No, because then we might as well rename nop to xchg e/rax,e/rax then

but it will disassemble differently now
